data/campaigns/Descent_Into_Darkness/scenarios/12_Endless_Night.cfg

```diff
@@ -10,68 +10,103 @@
+                    story= _ "The lich sometimes wonders if his endeavors are naught but a fruitless crusade, but then again, so what? Life, unlife, death... nothing has ever given his existence meaning. And yet, he still persists on? For what?"
+                    {STORYTXT_BACKGROUND end.jpg}
+                [/part]
+                [part]
+                    story= _ "<i>Reason and purpose...</i>"
+                    {STORYTXT_BACKGROUND end.jpg}
+                [/part]
+                [part]
+                    story= _ "He needs none. Philosophy only begets misery. Violence and savagery have been the essence of his existence, the forces that once controlled his life, the ones unrelinquished in undeath. Better to simply kill, without deliberation, without mercy, for what use is there to dwell on any alternative? He is not one for regrets. He has never been."
+                    {STORYTXT_BACKGROUND end.jpg}
+                [/part]
+                [part]
+                    story= _ "Rumors circulate of a lich that preys on scouting patrols. A hero, gathering his loyal troops to him, decides to put an end to the evil."
+                    {STORYTXT_BACKGROUND end.jpg}
                 [/part]
             [/then]
         [/if]
+        # These if statements are before the prestart event, so are off-by-one compared to the other checks of timesForever
         [if]
             {VARIABLE_CONDITIONAL timesForever numerical_equals 1}
             [then]
                 [part]
-                    story= _ "Mal Keshar continues to study the book, and re-reads the final entry."
+                    story= _ "Mal Keshar continues to study the book and re-reads the final entry."
                     {STORYTXT_BACKGROUND end.jpg}
                 [/part]
                 [part]
                     # po: This text comes from the epilogue of Secrets of the Ancients, edited to avoid revealing the name of the campaign or the names of the protagonists
-                    story = _"<i>“I realize now... the world is not ready for our knowledge. I give up on all of them. I am going to live in the old troll city and learn the applications of mountain fire. In some future time, less conservative attitudes will dominate, and I can certainly afford to wait. In the meantime, we are doing what we can to hasten that day.”</i>"
+                    story = _"<i>“I realize now... the world is not ready for our knowledge. They cannot understand it, that primordial fear... we have conquered it, death, but they look upon us and superficial judgment drives them to ignorant terror. For now, I shall just with draw, perhaps live in the old troll city and learn the applications of mountain fire. In some future time, less conservative attitudes may dominate. Maybe then, our knowledge can be put to good use. Until then, I can certainly afford to wait. In the meantime, we are doing what we can to hasten that day.”</i>"
```

Comment:
"withdraw" should be a single word

Reply:
I don't have a problem with your word choices, but that second sentence is long and grammatically questionable. It also makes the first part of Ardryn-na's journal entry in SotA redundant. I reworked the SotA the journal entry to be closer to what you have in the PR:

_We prevailed today on the field, but our victory did not please me. I realize now... the world is not ready for our knowledge. They cannot understand that we have conquered that primordial fear: death. Instead, as they look upon us, judgment of our appearance drives them to ignorant terror. I give up on all of them. For now, I shall just withdraw — perhaps live in the old troll city and learn the applications of mountain fire. In some future time, less conservative attitudes may dominate, and our knowledge can be put to good use. I can certainly afford to wait. In the meantime, we are doing what we can to hasten that day._

Will this work for you? I did put back the line about giving up on everyone because I feel like it's important for my character, but you can skip it with an ellipsis if you don't feel like it fits DiD. In fact, you can skip the bit about the troll city too. I also changed the order of the sentences so that there weren't five sentences in a row starting with an adverbial time phrase.

Reply:
> The second sentence was more stream-of-conscious-like writing, but I suppose that's less fitting for a journal.
Not necessarily, but I didn't use it anywhere else, so it would be in this case.
Can you add the change to the SotA epilogue to this PR so both quotes get changed at the same time?

Reply:
Done, if you have any other concerns, please let me know. :)